hi-ml/src/health_ml/utils/box_utils.py

```diff
@@ -2,6 +2,7 @@
 from typing import Optional, Sequence, Tuple
```

Comment:
Could you add the Microsoft header here?

Reply:
If these files are missing the header, I suspect many others will too. Do you mind if we address that issue in a different PR?

Reply:
I think we could address it where we spot, what do you think?

Reply:
Done.